mesonbuild/modules/qt5.py

```diff
@@ -23,6 +23,4 @@ def __init__(self, interpreter):
 
 
 def initialize(*args, **kwargs):
-    mlog.warning('rcc dependencies will not work reliably until this upstream issue is fixed:',
```

Comment:
`mlog` is not used anymore in this module, so the import can be removed. That's what the lgtm warning is about.